theano/scan_module/scan_utils.py

```diff
@@ -185,6 +185,8 @@ def clone(output,
         as the original graph. If False, clone them. Note that cloned
         shared variables still use the same underlying storage, so they
         will always have the same value.
+
+    :param copy_inputs: deprecated, use share_inputs.
```

Comment:
I think it's better to not document deprecated args.  They are only there to keep old code working.